app/views/dashboards/show.html.erb

```diff
@@ -29,7 +29,7 @@
               <td><%= link_to submission.title, work_path(submission) %></td>
               <td><%= submission.depositor.sunetid %></td>
               <td><%= submission.collection.name %></td>
-              <td><%= submission.state.humanize %></td>
+              <td><%= render Works::StateDisplayComponent.new(work: submission) %></td>
```

Comment:
This is part of @mjgiarlo's efforts to normalize how we display state info.  He will have a PR forthcoming after _this_ PR is merged to finish this work.